ql/src/java/org/apache/hadoop/hive/ql/io/parquet/timestamp/JulianDate.java

```diff
@@ -0,0 +1,304 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// This class is ported from org.jodd:jodd-util:6.0.0 and remove unreferenced code
```

Comment:
documented where the code comes from, and also preserved the original license header